operators/src/processing/expression/mod.rs

```diff
@@ -993,7 +908,7 @@ mod tests {
 
         assert_eq!(
             res,
-            [None, Some(6), None, Some(12), Some(15), Some(18)] // first None is because 1+1+1 == no_data_value, second None is because all inputs are masked because 3 == no_data_value
+            [Some(3), Some(6), None, Some(12), Some(15), Some(18)] // third is None is because all inputs are masked because 3 == no_data_value
```

Comment:
why has it changed?

Reply:
Referring to the docs, we cannot compute data that get's no data by accident.